docs/UserGuide.md

```diff
@@ -261,6 +261,39 @@ Examples:
 * If there are one or more associated schedules associated with the 2nd session in the Session List, 
   `list` followed by `sdel 2 f/ true` will delete all schedules associated with the 2nd session, then delete the session 
   itself
+  
+### Filtering Sessions by time: `sview`
+You can filter the Session List by time period to manage your sessions.
+
+Format: `sview p/PERIOD`
+ * Filters the Session List to only display those within the specified period.
+ * Right above the Session List, you can see the filter currently being applied.
+ * The recognized periods are as follows:
+ 
+ | Period | Sessions displayed |
+ | -------- | -------- |
+ | all| All sessions (including past ones)|
+ | future | All sessions that start from current date onwards|
+ | `+[x][unit]` | Sessions within next x time units (inclusive of current date)|
+ | `-[x][unit]` | Sessions within past x time units (inclusive of current date)|
+ 
+ * The recognised units are as follows:
+
+ | Unit | Time unit parsed |
+ | -------- | -------- |
+ | d | day |
+ | w | week|
+ | m | month|
+ | y | year|
+*case insensitive
+ 
+Examples:
+*  `sview /all` Displays all sessions.
+*  `sview /+0D` Display all sessions today.
+*  `sview /-1D` Display sessions within the past 2 days (yesterday and today).
+*  `sview /+2w` Display sessions within the next 2 weeks 
+(e.g. If today is Friday, display from today til the Thursday that falls 2 weeks later).
```

Comment:
(e.g. If today is Friday, display from today **till** the **Friday** that falls 2 weeks later).
In other words, it will display today and 2 weeks (14 days) in the future